packages/sdk/contractkit/src/kit.ts

```diff
@@ -275,14 +268,13 @@ export class ContractKit {
   }
 
   async getFeeCurrencyWhitelist(): Promise<StrongAddress[]> {
-    const [whitelist, celo] = await Promise.all([
+    const [whitelist] = await Promise.all([
       this._web3Contracts.getContract(CeloContract.FeeCurrencyWhitelist),
-      this.contracts.getGoldToken(),
     ])
     const feeCurrencyWhitelist = (await whitelist.methods.getWhitelist().call()) as StrongAddress[]
 
-    // Making sure we always have at least CELO and a uniq list.
-    return [...new Set([celo.address, ...feeCurrencyWhitelist])].sort()
+    // Making sure we always have a uniq list.
+    return [...new Set(feeCurrencyWhitelist)].sort()
```

Comment:
i love sets but i dont get why we are using one here. are we worried the whitelist from the contract will have duplicates?

and why do we need to sort the addresses?

Reply:
doesn't hurt and makes sure our tests snapshots stay the same.

Reply:
Hey reading over this again. Id like to propose that instead of adding a method to `kit` we instead create a contractWrapper. for FeeCurrencyWhitelist 

a) its the only core contract without a Wrapper
b) I think keeping `kit` very small in scope is a good idea.

Reply:
I would include  `getFeeCurrencyInformation` on the wrapper too

Reply:
Alright.